dev-util/smartgit/smartgit-6.5.6.ebuild

```diff
@@ -23,7 +23,7 @@ RESTRICT="fetch"
 RDEPEND=">=virtual/jre-1.7:1.7"
```

Comment:
No worries, that is why we have travis and doing a review!

Just add `hg` and `git` into `IUSE` and something like `hg? ( dev-vcs/mercurial )` to `RDEPEND`.